### ISSUE TITLE
Pasting a URL into a blank block turns it into an embed, even if it's non-embeddable.

### ISSUE THREAD

**Issue by mintplugins** [WordPress/gutenberg#9450]
**Describe the bug**
In Gutenberg version: 3.6.2, pasting a URL into a blank block turns the block into an embed block, even if it's non-embeddable. 

**To Reproduce**
1. Copy "https://google.com" to your clipboard.
2. In Gutenberg, type some text in a paragraph block, then hit enter, creating a new block.
3. Use your keyboard's hotkeys to paste the URL
3. The block automatically turns into an embed block.

**Expected behavior**
I expect that pasting a URL keeps the block type as-is. If it's a paragraph block type, it should remain as one, even if I paste a URL, especially if that URL is not embeddable.

**Screenshots**
<img width="661" alt="screen shot 2018-08-29 at 7 00 01 pm" src="https://user-images.githubusercontent.com/1638128/44820217-e4c3b580-abbd-11e8-8904-d0275fe5345e.png">

**Desktop (please complete the following information):**
 - OS: OSX
 - Browser [e.g. chrome, safari]Chrome
 - Version 68.0.3440.84

**Additional context**
Gutenberg version: 3.6.2

**Comment by youknowriad** [comment]
I recall a PR or an issue about this @notnownikki? Is this a duplicate?

**Comment by designsimply** [comment]
Possibly, #4799?

**Comment by youknowriad** [comment]
Ok Let's close as duplicate and continue work there.

**Comment by jdaviescoates** [comment]
I've looked at the so-called duplicates, but none of them actually address the issue I am facing and this issue seems like the closest one, so commenting here for now instead of creating a new issue:

I find it very frustrating that it is seemingly impossible to simple paste a URL into a post without it becoming an embed.

On this page:
http://www.ctrlshiftsummit.org.uk/partners/

I've had to manually type out lots of our Partner website URLs because if I don't it become and uneditable/ non-convertible block and not just a pasted URL which is what I want.

Sometimes the embed fails and I am them presented with the option to "covert to a link" which is useful - but I want that option to be available in ALL cases.

I like it when e.g. Twitter links automatically get turned into embeds, but in most cases I simply want to paste a URL and for it to NOT be an embed.

With Gutenberg doing this exceedingly simple task has become arduous, which is rather frustrating to say the least.

In https://github.com/WordPress/gutenberg/issues/9451 it says the convert option is not available for embeds because it wouldn't always work - but surely it could always simply be converted to a link, no?

In short: please please make it possible to simply be able to copy/ paste URLs into posts/ pages in Gutenberg.  Thanks!
